app/routes/overview/components/Overview.tsx

```diff
@@ -96,6 +96,18 @@ const Overview = () => {
     [articles, articlesToShow, pinned]
   );
 
+  const scrollToTop = () => {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+
+    setTimeout(function () {
+      setArticlesToShow(ARTICLES_TO_SHOW);
+    }, 200);
+
+    setTimeout(function () {
+      setEventsToShow(EVENTS_TO_SHOW);
+    }, 200);
```

Comment:
Why are these here? Not sure how they are related

Reply:
I think they kicked in somehow before the scroll function which led to not reaching the top when applying the scroll function but the bottom of the reduced event list. Adding the timeout fixed the problem

Reply:
Could you try without these? When I'm running it locally it worked just fine without

Reply:
Or what environment did you use where it didn't work? if they are indeed necessary it'd be nice with @Bestem0r 's suggestion of adding them to the same setTimeout function, and maybe adding a comment as to why they are required as it doesn't feel super intuitive
```suggestion
    setArticlesToShow(ARTICLES_TO_SHOW);
    setEventsToShow(EVENTS_TO_SHOW);
```
or
```suggestion
    // This timeout is needed because of bla bla bla
    setTimeout(() => {
      setArticlesToShow(ARTICLES_TO_SHOW);
      setEventsToShow(EVENTS_TO_SHOW);
    }, 200);
```

Reply:
Yeah I tried both in chrome and safari and for me it still doesn't work without the timeout, however I have now put them into one single timeout, and I added a comment to explain why they are needed